source/simulator/parameters.cc

```diff
@@ -2429,9 +2427,10 @@ namespace aspect
 
 
   template <int dim>
-  void Simulator<dim>::declare_parameters (ParameterHandler &prm)
+  void Simulator<dim>::declare_parameters (ParameterHandler &prm, const unsigned int mpi_rank)
   {
-    Parameters<dim>::declare_parameters (prm);
+    Parameters<dim>::declare_parameters (prm, mpi_rank);
+    VolumeOfFluidHandler<dim>::declare_parameters(prm);
```

Comment:
Is this an unrelated change, along with the one to line 1526? Not wrong, just curious.

Reply:
Yes, this line was for some reason hidden in ``Parameters::declare_parameters()``. I think this spot makes a lot more sense.